skeleton.js

```diff
@@ -0,0 +1,16 @@
+var scraper = require('website-scraper');
+
+scraper.scrape({
+	urls: ['https://stat.qa.wordpress.boston.com/2016/04/04/stat-dataviz-skeleton-post/'],
+	directory: 'app/assets/',
+	subdirectories: [
+		{directory: 'img', extensions: ['.jpg', '.png', '.svg']},
+		{directory: 'js', extensions: ['.js']},
+		{directory: 'css', extensions: ['.css']}
+	 ],
```

Comment:
This is looking/working great so far. Can we change the above subdirs to 'vendor/stat/img', 'vendor/stat/js', etc.. This way we can tell Natalia to ignore anything that's under a vendor subdir. These files will just be there for the purposes of rendering the skeleton index.html page. Also, Brunch by convention doesn't do any additional processing on assets under vendor dirs.